domain/src/main/java/in/koreatech/koin/domain/usecase/dining/GetDiningUseCase.kt

```diff
@@ -2,22 +2,14 @@ package `in`.koreatech.koin.domain.usecase.dining
 
 import `in`.koreatech.koin.domain.model.dining.Dining
 import `in`.koreatech.koin.domain.repository.DiningRepository
-import `in`.koreatech.koin.domain.repository.TokenRepository
 import javax.inject.Inject
 
 class GetDiningUseCase @Inject constructor(
     private val diningRepository: DiningRepository,
-    private val tokenRepository: TokenRepository
 ) {
     suspend operator fun invoke(date: String): Result<List<Dining>> {
-        if(tokenRepository.getAccessToken() == null) {  // 익명
-            return kotlin.runCatching {
-                diningRepository.getDining(date)
-            }
-        } else {    // 로그인 유저
-            return kotlin.runCatching {
-                diningRepository.getAuthDining(date)
-            }
+        return kotlin.runCatching {
```

Comment:
```suggestion
        return runCatching {
```
kotlin 지워도 좋을 것 같네요~